static/js/portico/tippyjs.js

```diff
@@ -0,0 +1,18 @@
+import $ from "jquery";
+import tippy from "tippy.js";
+
+function initialize() {
+    tippy("[data-tippy-content]", {
+        // Same defaults as set in web tippyjs module.
+        maxWidth: 300,
+        delay: [100, 20],
+        animation: true,
```

Comment:
We have `animation: false` in the web codebase; moved this to the bottom section.